tests/fixtures/ember-container-query-glint/output/src/components/widgets/widget-1.ts

```diff
@@ -2,10 +2,6 @@ import templateOnlyComponent from '@ember/component/template-only';
 
 interface WidgetsWidget1Signature {
   Args: {};
-  Blocks: {
-    default: [];
-  };
-  Element: null;
```

Comment:
**TODO**

These deleted lines show that, if the codemod is run on a project that already has the correct Glint signatures, the codemod will incorrectly remove `Blocks` and `Element`.

Just something for end-developers to keep in mind, until the codemod is updated to add `Blocks` and `Element`.